crates/astria-composer/src/executor/mod.rs

```diff
@@ -103,6 +104,29 @@ pub(super) struct Executor {
     max_bytes_per_bundle: usize,
 }
 
+#[derive(Clone)]
+pub(super) struct Handle {
+    serialized_rollup_transactions_tx: mpsc::Sender<SequenceAction>,
+}
+
+impl Handle {
+    pub(super) fn new(serialized_rollup_transactions_tx: mpsc::Sender<SequenceAction>) -> Self {
```

Comment:
This constructor should not be `pub(super)` - it should only be callable from inside this module

(Very minor nit: I don't think this needs an explicit constructor - just create the handle in `Executor::new` (or however it's constructor is called).)

Reply:
i think thats a good point. i ll try just creating it in the Executor::new